tgui/packages/tgui/interfaces/MarionetteRemote.js

```diff
@@ -0,0 +1,163 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Button, Flex, LabeledList, Section, Stack } from '../components';
+
+export const MarionetteRemote = (_props, context) => {
+  const { act, data } = useBackend(context);
+  const { entered_data, selected_command, implants } = data;
```

Comment:
`MarionetteRemoteType.ts` would allow for typing `entered_data, selected_command, implants`

Also, here you wouldn't need to declare implants.